Library/OcBootManagementLib/OcBootManagementLib.c

```diff
@@ -525,26 +536,49 @@ OcRunFirmwareApplication (
 }
 
 EFI_STATUS
-OcLaunchAppleBootPicker (
-  VOID
+OcGetAppleFirmwareUI (
+  IN APPLE_FIRMWARE_USER_INTERFACE_PROTOCOL  **UIProtocol,
+  IN APPLE_FIRMWARE_UI_VARS                  **UIVars OPTIONAL
```

Comment:
Aren't both only `OUT`?

Reply:
Yes, doh. Though with BASE_CR change this is now deleted.